applications/accounts/deploy/values.yaml

```diff
@@ -34,6 +34,8 @@ harness:
       value: "user"
     - name: DB_PASSWORD
       value: "password"
+    - name: JAVA_OPTS
+      value: -server -Xms64m -Xmx1024m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true  --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED
```

Comment:
would set the maximal heap size 10-20% lower than memory set in limit request to reserve some space for other processes

Reply:
fine for me, see new commit